server/journeys/screen/screen/controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { BaseJourneyController } from '../../base/controller'
+
+export class ScreenController extends BaseJourneyController {
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const saferCustodyOutcomeItems = await this.getReferenceDataOptionsForRadios(
+      req,
+      'outcome-type',
+      req.journeyData.saferCustodyScreening?.outcomeType || '',
+    )
+
+    res.render('screen/screen/view', {
+      saferCustodyOutcomeItems,
+      reasonForDecision: req.journeyData.saferCustodyScreening!.reasonForDecision,
```

Comment:
Similar to above, this needs to be
```
reasonForDecision:
        res.locals.formResponses?.['reasonForDecision'] || req.journeyData.saferCustodyScreening?.reasonForDecision,
```
to retain what the user entered in the event of a validation fail